_posts/2020-02-04-planning-ahead-for-v1.0.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "What would be kadalu v1.0?"
```

Comment:
title: "How are we building for Kadalu v1.0?"

Reply:
Done